webapi/src/main/java/org/sakaiproject/webapi/controllers/RubricsRestController.java

```diff
@@ -370,7 +371,9 @@ public ResponseEntity<byte[]> getPdf(@PathVariable String siteId, @PathVariable
         checkSakaiSession();
 
         ContentDisposition contentDisposition = rubricsService.getRubric(rubricId).map(rubric -> {
-            return ContentDisposition.builder("attachment").filename(rubric.title + ".pdf").build();
+            String filename = StringUtils.trimToEmpty(rubric.title).replace(".", "_");
+            filename = StringUtils.isNotBlank(filename) ? filename : "_";
+            return ContentDisposition.builder("attachment").filename(String.format("%s.pdf", filename)).build();
```

Comment:
I, personally, think using format on such short concatenations doesn't really help with readability. The compiler optimises these concatenations anyway, it replaces them with StringBuilder calls. Just my opinion though, nothing more :)

Reply:
Okay, I'll keep it in mind. For next time ;)